docs/faq.md

```diff
@@ -206,3 +206,12 @@ resource.customizations: |
       hs.message = "Controller doesn't report resource status"
       return hs
 ```
+Or it can be defined via `resource.customizations.health.<group_kind>` sub key.
```

Comment:
Please delete the deprecated settings style since we want to stop supporting it. Users will be able to find it using v2.0 docs version at https://argo-cd.readthedocs.io/en/stable/